snowflake_blueprints/errors.py

```diff
@@ -0,0 +1,15 @@
+EXIT_CODE_UNKNOWN_ERROR = 3
+EXIT_CODE_INVALID_CREDENTIALS = 200
+EXIT_CODE_INVALID_ACCOUNT = 201
+EXIT_CODE_INVALID_WAREHOUSE = 202
+EXIT_CODE_INVALID_DATABASE = 203
+EXIT_CODE_INVALID_SCHEMA = 204
+
+# Download Errors
+EXIT_CODE_INVALID_QUERY = 205
+EXIT_CODE_NO_RESULTS = 206
+
+# Upload Errors
+EXIT_CODE_FILE_NOT_FOUND = 205
+EXIT_CODE_INVALID_UPLOAD_VALUE = 206
+EXIT_CODE_INVALID_UPLOAD_COLUMNS = 207
```

Comment:
The first two overlap with the download errors.

Reply:
Yep. That was the existing behavior, and why I brought up the topic in #product-discussion. I decided to leave it as it was because it isn't harming anything.